docker/volumeplugin/exec.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	statusSuccess      = "Success"
-	statusFailure      = "Failed"
```

Comment:
Oh, I think K8s has a typo bug here. Since everywhere they use "Failure", not "Failed" for the status. Here is the test: https://github.com/kubernetes/kubernetes/blob/b359034817685a8d25bb51bae765308d9d200da0/pkg/volume/flexvolume/flexvolume_test.go#L102, and here is the documentation: https://github.com/kubernetes/kubernetes/tree/adef589e371bbbf1168374ade0763d35c9f9ee53/examples/volumes/flexvolume, and here is the example: https://github.com/kubernetes/kubernetes/blob/adef589e371bbbf1168374ade0763d35c9f9ee53/examples/volumes/flexvolume/lvm#L57.

They have not found this bug, is because the constant "Failed" never gets used anywhere in the code verify against the cmd failure.